api-reference/10 UI Components/dxOverlay/1 Configuration/copyRootClassesToWrapper.md

```diff
@@ -5,7 +5,27 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Copies user-defined classes from the root element to the wrapper element. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+The following markup illustrates relations between wrapper and root elements when you set **copyRootClassesToWrapper** to **true**:
```

Comment:
In which cases this property should be enabled?

Reply:
`relations` is not the right word. Do you mean 'relationship'?